zap/src/main/java/org/parosproxy/paros/core/scanner/Scanner.java

```diff
@@ -103,6 +105,7 @@
     private User user = null;
     private TechSet techSet;
     private Set<ScriptCollection> scriptCollections = new HashSet<ScriptCollection>();
+    private Set<ScanFilter> scanFilters = new LinkedHashSet<>();
```

Comment:
Why a `Set`? Is it to prevent duplicates?

Reply:
I could have used ArrayList/Array but though of going with set as we are not accessing with index. but not sure much which one to go with, so went ahead with set , are there any concerns regarding using set here ?

Reply:
It's more about the intent/meaning of that collection than performance reasons.

Reply:
so you are recommending to change it to List ? or is it ok ?

Reply:
I'd prefer a `List`, that matches better the meaning we want here.

Reply:
sure will do the changes.